internal/webserver/server.go

```diff
@@ -307,7 +307,7 @@ func (webserver *WebServer) ConfigureStandardRoutes() {
 //
 // Trigger
 //
-// Available when HTTPTrigger is specified as the binding in configuration. This API
+// Available when 'http'' is specified as the Trigger Type in configuration. This API
```

Comment:
"http"? may want to fix the quotation here

Reply:
Fixed and forced pushed.